index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
+              connector).
+            </p>
+            <p>
+              The <a><code>latency</code></a> attribute value depends on the
+              platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>latency</code></a> attribute value will be
+              updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimestamp devicePosition
+          </dt>
+          <dd>
+            <p>
+              Holds an <a><code>AudioTimestamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a href=
+              "#widl-AudioTimestamp-contextTime"><code>contextTime</code></a>
+              member contains estimation of the audio stream position which is
+              currently being rendered by the
+              <a><code>AudioDestinationNode</code></a> (i.e. the current output
+              device audio stream position), in the same units and origin as
+              context's <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>;
+              the <a href=
+              "#widl-AudioTimestamp-performanceTime"><code>performanceTime</code></a>
+              member contains performance time estimation of the moment when
+              the stored audio stream position is rendered, in the same units
+              and origin as <code>performance.now()</code> (described in
+              [[!hr-time-2]]).
+            </p>
+            <p>
+              If the context's rendering graph has not yet processed a block of
+              audio, then <a><code>devicePosition</code></a> holds an
+              <code>AudioTimestamp</code> instance with both members containing
+              zero.
+            </p>
+            <p>
+              The value of <a><code>devicePosition</code></a> attribute can be
+              used to get performance time estimation for the slightly later
+              context's time value:
+            </p>
+            <pre class="highlight example">
+            function outputPerformanceTime(contextTime) {
+                var timestamp = destNode.devicePosition;
+                var elapsedTime = contextTime - timestamp.contextTime;
+                return timestamp.performanceTime + elapsedTime * 1000;
+            }
+            </pre>
+            <p>
+              In the above example the preciseness of the estimation depends on
+              how close the argument value is to the current output audio
+              stream position: the closer given <code>contextTime</code> to
+              <code>timestamp.contextTime</code>, the better preciseness of the
+              obtained estimation.
```

Comment:
Grammar:  "the closer given..., the better preciseness...." ->
"the closer the given contextTime is to timestamp.contextTime, the better the accuracy of the obtained estimation."